src/axelrod_fortran/strategies.py

```diff
@@ -1,73 +1,349 @@
-axelrod_python_classes = {
-    'grasr': None,
-    'k31r': None,
-    'k32r': None,
-    'k33r': None,
-    'k34r': 'Grudger',
-    'k35r': None,
-    'k36r': None,
-    'k37r': None,
-    'k38r': None,
-    'k39r': None,
-    'k40r': None,
-    'k41r': None,
-    'k42r': None,
-    'k43r': None,
-    'k44r': None,
-    'k45r': None,
-    'k46r': 'Eatherley',
-    'k47r': None,
-    'k48r': None,
-    'k49r': None,
-    'k50r': None,
-    'k51r': None,
-    'k52r': None,
-    'k53r': None,
-    'k54r': None,
-    'k55r': None,
-    'k56r': None,
-    'k57r': 'Nydegger',
-    'k58r': None,
-    'k59r': 'RevisedDowning',
-    'k60r': None,
-    'k61r': 'Champion',
-    'k62r': None,
-    'k63r': None,
-    'k64r': None,
-    'k65r': None,
-    'k66r': None,
-    'k67r': None,
-    'k68r': None,
-    'k69r': 'Joss',
-    'k70r': None,
-    'k71r': None,
-    'k72r': None,
-    'k73r': None,
-    'k74r': None,
-    'k74rxx': None,
-    'k75r': None,
-    'k76r': 'Tester',
-    'k77r': 'Feld',
-    'k78r': None,
-    'k79r': None,
-    'k80r': None,
-    'k81r': None,
-    'k82r': None,
-    'k83r': None,
-    'k84r': None,
-    'k85r': None,
-    'k86r': 'Grofman',
-    'k87r': None,
-    'k88r': None,
-    'k89r': None,
-    'k90r': None,
-    'k91r': None,
-    'k92r': None,
-    'k93r': 'UnnamedStrategy',
-    'kpavlovc': 'WinStayLoseShift',
-    'krandomc': 'Random',
-    'ktf2tc': 'TitFor2Tats',
-    'ktitfortatc': 'TitForTat',
+classifiers = {
```

Comment:
Naming things is difficult....

I considered both 'information' and 'details' but thought they were too vague and insufficiently descriptive. However, I also agree with the objection to 'classifiers'!